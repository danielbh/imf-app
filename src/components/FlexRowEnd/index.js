/**
 * Created by danielhollcraft on 9/15/17.
 */
import styled from 'styled-components';

const FlexRowEnd = styled.div`
  margin: 0; 
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-flex-flow: row wrap;
  justify-content: flex-end;
`;

export default FlexRowEnd;