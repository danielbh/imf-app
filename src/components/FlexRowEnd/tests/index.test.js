import React from 'react';
import { shallow } from 'enzyme';
import FlexRowEnd from '../index'

describe('<FlexRowEnd />', () => {
  it('matches snapshot', () => {
    expect(shallow(<FlexRowEnd />)).toMatchSnapshot();
  });
});