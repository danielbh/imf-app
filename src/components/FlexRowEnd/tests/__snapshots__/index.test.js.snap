// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<F /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;

exports[`<Fle /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;

exports[`<Flex /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;

exports[`<FlexRow /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;

exports[`<FlexRowEnd /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;

exports[`<RangeButtons /> matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="sc-bdVaJa bCmqOv"
/>,
  "nodes": Array [
    <div
      className="sc-bdVaJa bCmqOv"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <styled.div />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "bCmqOv",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="sc-bdVaJa bCmqOv"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="sc-bdVaJa bCmqOv"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.div />,
}
`;
